app/controllers/provider_foreman_controller.rb

```diff
@@ -63,11 +63,21 @@ def new
     assert_privileges("provider_foreman_add_provider")
     @provider_cfgmgmt = ManageIQ::Providers::ConfigurationManager.new
     @provider_types = ["Ansible Tower", ui_lookup(:ui_title => 'foreman')]
+    @server_zones = []
```

Comment:
This can be simplified to offload the work to postgres creating fewer ruby objects and improving performance as follows:
```ruby
@server_zones = Zone.create!(:name => "empty", :description => "nothing to see here")
=> [["Default Zone", "default"], ["nothing to see here", "empty"]]
```